web/.envprod

```diff
@@ -0,0 +1,2 @@
+REACT_APP_SERVER_URI="https://gregoryguterman.com"
+REACT_APP_SERVER_PATH="/deuces/socket.io"
```

Comment:
React has special handling for `.env.production` and `.env.development` that loads the correct env vars based on the environment.